go.mod

```diff
@@ -103,6 +103,9 @@ require (
 	github.com/garyburd/redigo v1.6.0 // indirect
 	github.com/ghodss/yaml v1.0.0 // indirect
 	github.com/go-errors/errors v1.0.1 // indirect
+	github.com/go-git/gcfg v1.5.0 // indirect
```

Comment:
What caused all the changes in `go.mod`? It looks like all of them are `indirect` dependencies, but none of the direct dependencies seem to have changed.

Reply:
Not sure what caused this. I tried removing it and that caused tests to fail stating that I should run `go mod tidy`. I ran `go mod tidy` which added this dependency back.